pr_template.md

```diff
@@ -0,0 +1,32 @@
+Resolves **[LINK ISSUE HERE]**
+
+### Description
+Please explain the changes you made here.
+
+### Acceptance Criteria
+- [ ] Code compiles correctly
+- [ ] Unit and/or integration tests have been added
+- [ ] A manual end-to-end test was conducted locally, ensuring the user experience isn't broken
+- [ ] ...
```

Comment:
I think it could be useful to keep something in here for whether this is behind a feature toggle, but not a strong opinion.

Reply:
Sounds good! I added it.